code/datums/mutantraces.dm

```diff
@@ -2131,14 +2164,61 @@ ABSTRACT_TYPE(/datum/mutantrace)
 			src.mob.kickMessage = "stomps"
 			src.mob.traitHolder?.addTrait("hemophilia")
 
+			src.mob.vis_contents += src.distort_under
+			src.mob.vis_contents += src.distort_suit
+			src.mob.vis_contents += src.distort_belt
+			src.mob.vis_contents += src.distort_satchel
+			src.mob.vis_contents += src.mask_gloves
+			src.mob.vis_contents += src.mask_backpack
```

Comment:
This and the subtraction can also just be edone eas `+=`ing a list instead of doing it one by one